From: Kefeng Wang <wangkefeng.wang@huawei.com>
Subject: pin_user_pages: fix underline length
Date: Fri, 7 Jun 2024 14:46:57 +0800

Documentation/core-api/pin_user_pages.rst:200: WARNING: Title underline too short.
folio_maybe_dma_pinned(): the whole point of pinning
===================================================

Link: https://lkml.kernel.org/r/20240607064657.681997-1-wangkefeng.wang@huawei.com
Signed-off-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/core-api/pin_user_pages.rst |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/core-api/pin_user_pages.rst~mm-remove-page_mkclean-fix
+++ a/Documentation/core-api/pin_user_pages.rst
@@ -197,7 +197,7 @@ INCORRECT (uses FOLL_GET calls):
     put_page()
 
 folio_maybe_dma_pinned(): the whole point of pinning
-===================================================
+====================================================
 
 The whole point of marking folios as "DMA-pinned" or "gup-pinned" is to be able
 to query, "is this folio DMA-pinned?" That allows code such as folio_mkclean()
_
